README.md

```
@@ -1,43 +1,76 @@
+5. **Go home**: Click the "hito" title to return to the home screen
+
+## Keyboard Shortcuts
+
+- `←` - Previous image
+- `→` - Next image
+- `Esc` - Close modal
+- `?` - Show/hide keyboard shortcuts overlay
+
+## Directory Structure
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)